MAUI/Carousel-View/Populating-Data.md

```diff
@@ -0,0 +1,275 @@
+---
+layout : post
+title: Populating Data in .NET MAUI Carousel View control | Syncfusion
+description: Learn here all about Populating Data support in Syncfusion .NET MAUI Carousel View (SfCarousel) control and more.
+platform : maui
+control : Carousel
+documentation : ug
+---
+
+# Populating Data in .NET MAUI Carousel View (SfCarousel)
+
+`SfCarousel` control, supports binding to different data sources such as IList Data Source, Observable Collection Data Source.
+
+## Through Binding
+
+Items can be populated in `SfCarousel` control through data source and applying custom template as explained below. 
+
+### Create a Model with Data
+
+`SfCarousel` items can be populated with a collection of image data. For example, a user may wants to create a `SfCarousel` control which will display a list of images.
```

Comment:
`SfCarousel` items can be populated with a collection of image data. For example, a user may want to create a `SfCarousel` control, which will display a list of images.

Reply:
Addressed the changes.